packages/fetchai/skills/erc1155_deploy/strategy.py

```diff
@@ -29,7 +29,7 @@
 DEFAULT_IS_LEDGER_TX = True
 DEFAULT_NFT = 1
 DEFAULT_FT = 2
-DEFAULT_ITEMS_IDS = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
```

Comment:
Note, token_ids cannot be set anymore, we autogenerate them. So I'd remove the constant and variable.

Reply:
What can be set is the number of tokens, which needs to be added instead.